### PR TITLE
DO-1981 Made the database template sensitive

### PR DESCRIPTION
**Pull request by francoislapierremessier**
Also refactored the CHANGELOG

### CODE REVIEW

File: templates/default/env/configs.erb [hunk 1]
```diff
@@ -10,6 +10,6 @@ PLUGINS="<%= @plugins.join(' ') %>"
 PROJECT_CREATED_NOTIFY_ADDRESS=<%= @project_created_notify_address %>
 PROJECT_DELETED_NOTIFY_ADDRESS=<%= @project_deleted_notify_address %>
 SECRET_SIDECAR_IMAGE=<%= @secret_sidecar_image %>
-SECRET_STORAGE_BACKEND=<%= @secret_storage_backend %>
+SECRET_STORAGE_BACKEND=<%= @secret_storage_backend || 'SecretStorage::DbBackend' %> <%# Having the key overrides the default value in the code even when empty %>
```
Comment (JonathanSerafini) [inline templates/default/env/configs.erb]: This line is waaaaaay too long ;)
Can you move the comment above

**Comment by francoislapierremessier** [comment]
@JonathanSerafini should be ready for a review now

File: templates/default/env/configs.erb [hunk 2]
```diff
@@ -10,6 +10,6 @@ PLUGINS="<%= @plugins.join(' ') %>"
...
 SECRET_TOKEN=<%= @secret_token %>
-
+MEMCACHE_SERVERS=<%= @memcache_servers %>
```
Comment (JonathanSerafini) [inline templates/default/env/configs.erb]: Plural ? Does that indicate the value could be comma separated values ? 
If so, perhaps Array(@memcache_servers).join(',') ?

**Review by JonathanSerafini — COMMENTED** [review]
Otherwise lgtm

**Comment by francoislapierremessier** [comment]
Made the requested changes plus changed the version bump to make it a minor one since there are functional differences

**Comment by JonathanSerafini** [comment]
mgowk

**Comment by francoislapierremessier** [comment]
that gif!

**Pull request merged**